Samples/Use legacy custom panel in 7.4/Resources/SamplePanelDefinition.cs

```diff
@@ -0,0 +1,27 @@
+using Litium.Foundation.Modules.ECommerce;
+using Litium.Web.Administration.Panels;
+
+namespace Litium.Accelerator.Mvc.Site.ECommerce.Panels
+{
+    /// <summary>
+    /// Defines panel definition, to display it in EcommerceArea.
+    /// </summary>
+    public class SamplePanelDefinition : PanelDefinitionBase<EcommerceArea, SamplePanelDefinition.SettingsModel>
```

Comment:
How is the SettingsModel used?

Reply:
They are described in out docs https://docs.litium.com/documentation/architecture/back-office_1/creating-custom-panel